lib/internal/crypto/cipher.js

```diff
@@ -164,7 +150,8 @@ Cipher.prototype.update = function update(data, inputEncoding, outputEncoding) {
   outputEncoding = outputEncoding || encoding;
 
   if (typeof data !== 'string' && !isArrayBufferView(data)) {
-    throw invalidArrayBufferView('data', data);
+    throw new ERR_INVALID_ARG_TYPE(
+      'data', ['string', 'Buffer', 'TypedArray', 'DataView'], data);
```

Comment:
@nodejs/crypto all other cases that validate the input like that convert strings to buffers before passing the data on. Is it intentional, that this is not done here?

Reply:
@BridgeAR Should we pull the `author ready` label until we get an answer for this question? Or are you comfortable landing as-is?

Reply:
I am comfortable to land this as-is. It's mainly about consistency and the behavior is not changed with this PR.